keps/sig-apps/3140-TimeZone-support-in-CronJob/README.md

```diff
@@ -175,17 +175,17 @@ to implement this enhancement.
 
 ##### Unit tests
 
-- `k8s.io/kubernetes/pkg/apis/batch/validation`: `2022-06-09` - `94.4%`
-- `k8s.io/kubernetes/pkg/controller/cronjob`: `2022-06-09` - `50.8%`
-- `k8s.io/kubernetes/pkg/registry/batch/cronjob`: `2022-06-09` - `61.8%`
+- `k8s.io/kubernetes/pkg/apis/batch/validation`: `2023-01-18` - `96.0%`
+- `k8s.io/kubernetes/pkg/controller/cronjob`: `2023-01-18` - `51.7%`
+- `k8s.io/kubernetes/pkg/registry/batch/cronjob`: `2023-01-18` - `56.3%`
 
 ##### Integration tests
 
 None.
 
 ##### e2e tests
 
-None.
+- [CronJob should support timezone](https://github.com/kubernetes/kubernetes/blob/db4c64b1a987675fff7a234e8633f4fd01c69530/test/e2e/apps/cronjob.go#L301): https://storage.googleapis.com/k8s-triage/index.html?sig=apps&test=should%20support%20timezone
```

Comment:
Apart from the unit tests, we only have a negative test for this right? It might be worth having a single positive e2e or integration test (eg that a cronjob with a valid TZ creates a job).

Reply:
I'm considering adding that, but that has a potential to be flaky, so I don't want to put it here as a hard requirement.